docs/error-codes.md

```
@@ -47,6 +47,12 @@
 |---|---|---|---|
 | `MEDICINE_001` | 404 | Medicine not found | 존재하지 않는 약물 |
 
+### Medication Schedule
+| 코드 | HTTP | 메시지 | 설명 |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)